electron/renderer.js

```diff
@@ -57,13 +58,25 @@ class Renderer {
         try {
             this.redirectOutputToConsole();
             mocha.setup({
-                ui: 'tdd'
+                ui: 'tdd',
             });
 
-            let mochaInst = new Mocha();
+            // Format the reporter options
+            let reporterOptions;
+
+            // Parse string as an object
+            if (typeof this.options.reporterOptions === "string") {
```

Comment:
why not create mocha first and then call
`mochaInst.reporter(reporterType, options)` ?  

```js
 if (typeof this.options.reporterOptions === "string") {
    let reporterOptions = querystring.parse(
        this.options.reporterOptions
    );
    mochaInst.reporter(this.options.reporter, reporterOptions);
}
```